frontend/src/components/BarChartWidget.tsx

```diff
@@ -151,122 +91,32 @@ const BarChartWidget = (props: Props) => {
-                                        props.significantDigitLabels,
-                                        "",
-                                        "",
-                                        columnMetadata,
-                                    );
-                                }}
-                            />
-                            {!props.hideLegend && (
-                                <Legend
-                                    verticalAlign="top"
-                                    onClick={toggleBars}
-                                    onMouseLeave={() => setBarsHover(null)}
-                                    onMouseEnter={(e: any) => setBarsHover(e.dataKey)}
-                                    formatter={RenderLegendText}
-                                />
-                            )}
-                            {props.bars.length > 0 &&
-                                props.bars.slice(1).map((bar, index) => {
-                                    return (
-                                        <Bar
-                                            dataKey={bar}
-                                            fill={colors[index]}
-                                            key={index}
-                                            fillOpacity={getOpacity(bar)}
-                                            hide={hiddenBars.includes(bar)}
-                                            stackId={props.stackedChart ? "a" : `${index}`}
-                                            isAnimationActive={false}
-                                        >
-                                            {!props.hideDataLabels &&
-                                                props.stackedChart &&
-                                                index === props.bars.length - 2 && (
-                                                    <LabelList
-                                                        position="right"
-                                                        valueAccessor={valueAccessor(bar)}
-                                                        formatter={(tick: any) => {
-                                                            return TickFormatter.stackedFormat(
-                                                                tick,
-                                                                xAxisLargestValue,
-                                                                props.significantDigitLabels,
-                                                                props.bars.slice(1),
-                                                                props.columnsMetadata,
-                                                            );
-                                                        }}
-                                                    />
-                                                )}
-                                            {!props.hideDataLabels && !props.stackedChart && (
```

Comment:
hideDataLabels is not used here. I think something is missing

Reply:
unsupported with the current library. it's being removed from the UI and component contract.